presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/query/PinotQueryGenerator.java

```diff
@@ -146,32 +146,53 @@ public PinotQueryGeneratorContext getContext()
         }
     }
 
-    public static class GeneratedPql
+    public enum PinotQueryFormat {
+        PQL,
+        SQL
+    }
+
+    public static class GeneratedPinotQuery
     {
         final String table;
+        final String sql;
```

Comment:
This may be a nitpick, so feel free to ignore it: Should we just have a 'final String query' field here along with a 'final PinotQueryFormat queryFormat', instead of having these two strings and then checking that only one of them is set ?

Reply:
I'm actually thinking of the same, just don't want to change this initially. Glad you bring this up and I can fix it in this iteration.